common/service/dynamicconfig/constants.go

```diff
@@ -93,6 +93,8 @@ var keys = map[Key]string{
 	DisableListVisibilityByFilter:         "frontend.disableListVisibilityByFilter",
 	FrontendThrottledLogRPS:               "frontend.throttledLogRPS",
 	EnableClientVersionCheck:              "frontend.enableClientVersionCheck",
+	EnableRawHistoryQuery:                 "frontend.enableRawHistoryQuery",
+	EnableRawHistoryQueryByDomain:         "frontend.enableRawHistoryQueryByDomain",
```

Comment:
nit: suffixing with "Query" can be confusing for the reader because Query has other meaning within workflows in cadence. Can we call this something else - one suggestion would be frontend.sendRawWorkflowHistory

Reply:
> nit: suffixing with "Query" can be confusing for the reader because Query has other meaning within workflows in cadence. Can we call this something else - one suggestion would be frontend.sendRawWorkflowHistory

done